src/controllers/tripRequests.js

```diff
@@ -68,10 +84,10 @@ export const createTripRequest = async (req, res) => {
     );
   }
   tripRequest.managerId = req.user.managerId;
-
+  let destinations = tripRequest.destinationsId;
+  delete tripRequest.destinationsId
   try {
-    const result = await tripServices.createTripRequest(tripRequest);
-
+    const trip = await tripServices.createTripRequest(tripRequest, destinations);
     return res.status(201).send("Trip request successfully created");
   } catch (err) {
     console.log(err);
```

Comment:
can you show user the error messages you didn't catch?

Reply:
consoling the err is not handling it. you need to properly handle it. it needs to be returned to the client in a proper way